.github/workflows/lint.yml

```diff
@@ -16,8 +16,7 @@ jobs:
           fetch-depth: 0
 
       - name: Run ansible-lint
-        # replace `main` with any valid ref, or tags like `v6`
-        uses: ansible/ansible-lint-action@v6
+        uses: ansible/ansible-lint@v6.22.0
```

Comment:
This is making some new complains about linting.. at least I see the logs from latest execution showing 2 errors. I would keep it separate from this PR.